test/unit/org/apache/cassandra/cql3/restrictions/ClusteringColumnRestrictionsTest.java

```diff
@@ -1669,7 +1669,7 @@ private static Restriction newMultiEq(TableMetadata tableMetadata, int firstInde
         TupleType tupleType = new TupleType(types);
         return new SimpleRestriction(ColumnsExpression.multiColumns(columns, tableMetadata),
                                      Operator.EQ,
-                                     Terms.of(new MultiElements.Value(tupleType, asList(values))));
+                                     Terms.of(new MultiElements.Value(tupleType, asList(values))), false);
```

Comment:
nit/formatting: I'd move these `false` arguments to the next line so they're on the same level of indentation as the other arguments.